elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAsyncConfiguration.java

```diff
@@ -105,4 +102,23 @@ public AsyncQueryDAO buildAsyncQueryDAO(Elide elide) {
         Elide asyncElide = new Elide(asyncElideSettings);
         return new DefaultAsyncQueryDAO(asyncElide, asyncElide.getDataStore());
     }
+
+    /**
+     * Configure the AsyncQueryDAO used by async query requests.
+     * @param elide elideObject.
+     * @return an AsyncQueryDAO object.
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    @ConditionalOnProperty(prefix = "elide.async", name = "defaultResultStorageEngine", matchIfMissing = true)
+    public ResultStorageEngine buildResultStorageEngine(Elide elide) {
+        // Creating a new ElideSettings and Elide object for Async services
+        // which will have ISO8601 Dates. Used for DefaultAsyncQueryDAO.
+        ElideSettings asyncElideSettings = new ElideSettingsBuilder(elide.getDataStore())
+                .withEntityDictionary(elide.getElideSettings().getDictionary())
+                .withISO8601Dates("yyyy-MM-dd'T'HH:mm'Z'", TimeZone.getTimeZone("UTC"))
+                .build();
+        Elide asyncElide = new Elide(asyncElideSettings);
+        return new DefaultResultStorageEngine(asyncElide, asyncElide.getDataStore(), "http://localhost:8080");
```

Comment:
`http://localhost:8080` should not be hardcoded. we should use `settings.getAsync().getBaseUrl()`;